src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -145,7 +146,8 @@ private Binder LookupSymbolsWithFallback(LookupResult result, string name, int a
         /// <summary>
         /// Look for symbols that are members of the specified namespace or type.
         /// </summary>
-        private void LookupMembersWithFallback(LookupResult result, NamespaceOrTypeSymbol nsOrType, string name, int arity, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo, ConsList<TypeSymbol> basesBeingResolved = null, LookupOptions options = LookupOptions.Default)
+        private void LookupMembersWithExtensionsAndFallback(LookupResult result, NamespaceOrTypeSymbol nsOrType, string name, int arity,
```

Comment:
>LookupMembersWithExtensionsAndFallback [](http://example.com/codeflow?start=21&length=38)

I am not sure why this method got renamed. Consider reverting the rename. Otherwise, we would have to rename pretty much every method in this file.